UMN-OSDFrontEnd/AppSettings.Example.json

```diff
@@ -83,6 +80,79 @@
     {
       "tabName": "TabApplications",
       "enabled": true
+    },
+    {
+      "tabName": "Example Custom Tab",
+      "tabType": "CustomTab",
+      "enabled": true,
+      "tabDisplayName": "Computer Setup",
+      "tabLayout": [
+        { // Example static text box
+          "groupTitle": "some title",
+          "itemType": "textBlock",
+          "dynamic": false,
+          "manualTextBlock": {
+            "text": "This is just example text"
+          }
+        },
+        { // Example dynamic text from ts variable
+          "groupTitle": "some title",
+          "itemType": "textBlock",
+          "dynamic": true,
+          "tsVariableTextBlock": {
+            "tsVariable": "tsvariablename"
+          }
```

Comment:
Not sure how this layout item is supposed to work. Including it causes System.NullReferenceException: 'Object reference not set to an instance of an object.' exception error